kolibri/core/auth/test/test_user_import.py

```diff
@@ -136,77 +137,85 @@ def test_create_user_not_exist_bad_username(self):
 ]
 
 
+def test_device_not_setup(self):
```

Comment:
Best to just wrap this in another test case class (you can do standalone functions for tests, but the form is a bit different).
```
class UserImportNoSetupTestCase(TestCase):
    def test_device_not_setup(self):
```